Sources/NetworkService/HTTPMethod.swift

```diff
@@ -0,0 +1,12 @@
+/// The method to use in the HTTP request. Indicates the method to be performed
+/// on the resource identified by the Request-URI.
+public enum HTTPMethod: String, CaseIterable {
```

Comment:
Why do we need the conformance to the `CaseIterable` protocol?

Reply:
I've moved it to test target.